packages/vscode/src/extension.ts

```diff
@@ -1,12 +1,19 @@
 import { AnalysisEngine } from "@githru-vscode-ext/analysis-engine";
 import * as vscode from "vscode";
-import { COMMAND_LAUNCH } from "./commands";
+import { COMMAND_GET_ACCESS_TOKEN, COMMAND_LAUNCH } from "./commands";
 import { findGit, getBaseBranchName, getBranchNames, getGitConfig, getGitLog, getRepo } from "./utils/git.util";
 import { mapClusterNodesFrom } from "./utils/csm.mapper";
 import WebviewLoader from "./webview-loader";
 
 let myStatusBarItem: vscode.StatusBarItem;
 
+const getGithubToken = async () => {
+    const configuration = vscode.workspace.getConfiguration();
+    const githubToken: string | undefined = configuration.get("githru.github.token");
+    
+    return githubToken;
+}
```

Comment:
함수 내부에 반환용 변수에 string | undefined 을 정의하는것보다
함수 리턴정의로 올리는게 읽기 좋을 듯 합니다.

```suggestion
const getGithubToken = async () : string | undefined => {
    const configuration = vscode.workspace.getConfiguration();
    const githubToken = configuration.get("githru.github.token");
    return githubToken;
}
```

또는


```suggestion
const getGithubToken = async () : string | undefined => {
    const configuration = vscode.workspace.getConfiguration();
    return configuration.get("githru.github.token");
}
```

Reply:
그렇네요, 더불어 async가 필요없어서 이 역시 제거하여 함수 코드를 좀 더 깔끔하게 작성해서 아래와 같이 수정하는건 어떨까요
`
const getGithubToken = () : string | undefined => {
    const configuration = vscode.workspace.getConfiguration();
    return configuration.get("githru.github.token");
}
`

Reply:
async는 저도 못봤네요 ㅎㅎㅎ 좋습니다 👍👍👍